src/server/handlers/services/{ServiceType}.ts

```diff
@@ -89,13 +104,13 @@ const get = async (_context: unknown, request: Request, h: ResponseToolkit): Pro
         serviceType,
         payload,
         span
-      )
-        .catch(err => {
-          // Do nothing with the error - forwardGetServicesServiceTypeRequestFromProviderService takes care of async errors
-          Logger.error('Services::put - forwardGetServicesServiceTypeRequestFromProviderService async handler threw an unhandled error')
-          Logger.error(ReformatFSPIOPError(err))
-        })
-
+      ).catch((err) => {
```

Comment:
```suggestion
      ).catch(err => {
```

Ok, there are a lot of these. This is the last one I am noting, but this comment is applicable everywhere else this appears.

I am surprised the linter was ok with this.